### ISSUE TITLE
Add existing Question in questionset archive_v1.1.0-beta.1/storage-and-long-term-preservation/long-term-preservation-datasets

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#65]
**Describe the request**  
Add Question about project owner / data owner on departure of the scientists.

**Propose what exactly should be changed**  
The question about the project partner asks for the contact person for data management. Since this can also be someone with a limited contract, here is a more specific question. Possibly change question at (archive_v1.1.0-beta.1/general/project-partners-partner/contact)?

**Proposed setting or content**   
*question text en*  
"~~Is there a~~ Who is the permanent project owner/data owner who can decide on the data after the researcher leaves? ~~It can happen that the data has to be migrated to a new system, for example after 3 years. Who should be contacted then (name, e-mail address, institute/university)?~~"
*help text en*
"A similar question is, what happens if the access data is lost or, for example, if the chair holder retires? This should be taken into consideration later on by means of a workflow."
*question text ger*
"~~Gibt es einen~~ Wer ist der dauerhafte~~n Projektowner/ Datenowner~~ Besitzer der Daten, der nach dem Weggang des Forschenden über die Daten entscheiden kann? ~~Es kann passieren, dass die Daten z.B. nach 3 Jahren in ein neues System migriert werden müssen. Wer ist dann zu kontaktieren (Name, E-Mail-Adresse, Institut/Universität)?~~"
*help text ger*
"Eine ähnliche Fragestellung ist, was passiert bei Verlust der Zugangsdaten oder beispielsweise beim Ausscheiden/Ruhestand des Lehrstuhlinhabers? Dies sollte später über einen Workflow mit bedacht werden."

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] key responsible_person_name
- [x] path storage-and-long-term-preservation/long-term-preservation-datasets/responsible_person_name
- [x] question text (en and ger)
- [x] help text (en and ger)
- [x] attribute /project/dataset/preservation/responsible_person/name
- [x] attribute /project/dataset/preservation/responsible_person/email
- [x] question storage-and-long-term-preservation/long-term-preservation-datasets/responsible_person/name
- [x] question storage-and-long-term-preservation/long-term-preservation-datasets/responsible_person/email

**Comment by jmfrenzel** [comment]
We need a new attribute data set level. This describes the person being responsible for the dataset  also after project end.

**Comment by jmfrenzel** [comment]
Note if dmp to be published contact details belong to sensitive data. This includes E-Mail.

**Comment by smohex** [comment]
UDE/TUDO: Agreement, introduce new attribute

**Comment by smohex** [comment]
added new attribute /project/dataset/preservation/responsible_person first and then the two attributes as mentioned above

**Comment by smohex** [comment]
Does the question actually require two attributes and then two questions? My suggestion as a question: Wer ist der dauerhafter Besitzer der Daten, der nach dem Weggang des Forschenden über die Daten entscheiden kann? Bitte geben Sie Name und E-Mail-Adresse an.
Wenn das auch in Ordnung wäre, würde lösche ich das bereits angelegte Attribut wieder!

**Comment by jmfrenzel** [comment]
We need two attributes ( and therefore two questions) to allow for easy (machine-readable) extraction of the data.

**Comment by smohex** [comment]
The two questions were inserted at the end of the question set "Long-term archiving"/"Langzeitarchivierung".
Keys: responsible_person_name AND responsible_person_email
Attributes and questions paths as specified

**Comment by ClaraRosalie** [comment]
These changes should be implemented in upstream as well.

**Comment by ClaraRosalie** [comment]
PR in upstream #111

**Comment by jmfrenzel** [comment]
We need to readopt the content from https://github.com/rdmorganiser/rdmo-catalog/pull/111

**Comment by ClaraRosalie** [comment]
I implemented the changes according to rdmorganiser#111 in #102.
